pages/configuration/configuration-settings.mdx

```diff
@@ -9,32 +9,182 @@ import { Steps } from 'nextra/components'
 
 # Configuration settings
 
-The main Memgraph configuration file is available in
-`/etc/memgraph/memgraph.conf`. You can modify that file to suit your specific
-needs. Additional configuration can be specified by including another
-configuration file, in a file pointed to by the `MEMGRAPH_CONFIG` environment
-variable or by passing arguments on the command line.
+Memgraph has a set of configuration options that can be fine-tuned for specific needs. 
 
-For example, you can start the `memgraph` Docker image with `docker run
-memgraph/memgraph --bolt-port=7687 --log-level=TRACE`.
+The main Memgraph configuration file is available at the
+`/etc/memgraph/memgraph.conf`. The file contains a set of default configuration values 
+and key-value pairs that can be modified to suit your specific needs.
 
-<Callout type="info">
+Each configuration setting is in the form: `--setting-name=value`.
 
-If you are working with Memgraph Platform Docker image, you should pass
-configuration flags inside of environment variables, e.g. `docker run -p
-7687:7687 -p 7444:7444 -p 3000:3000 -e MEMGRAPH="--bolt-port=7687
---log-level=TRACE" memgraph/memgraph-platform`.
+You can check the current configuration by using the following query:
 
-</Callout>
+```opencypher
+SHOW CONFIG;
+```
 
-Each configuration setting is in the form: `--setting-name=value`.
+## Changing configuration settings
+
+The `memgraph.conf` file is the persistency for configuration, and the way you
+can change the configuration settings depending on the way you are using Memgraph
+and configuration settings you want to change.
+
+Most of the configuration changes need to happen before Memgraph is started, 
+there is a set of configuration settings that can be changed during **runtime**. 
+
+### Change configuration settings on startup
+
+Changing the configuration settings for Memgraph differs if you are using
+Memgraph with **Docker**, **Docker Compose**, or if it was installed on the native **Linux**.
+
+<Tabs items={["Docker", "Docker Compose" , "Linux"]}>
+
+    <Tab>
+
+        You can customize the configuration by
+        passing configuration options within the `docker run` command.
+
+        When you are working with `memgraph` or `memgraph-mage` images, you should pass
+        configuration options as arguments.
```

Comment:
Is this part necessary anymore? It was important to note when there was an image where this was not true (platform).

Reply:
No, that it is important. We need to show how to change arguments to the`memgraph` and `memgraph-mage` images. This differs from the Native version and the Compose version. 

I have a feeling we do not understand each other on this one. 😄

Reply:
I understand Docker differs from native and Compose, but I probably meant it's not necessary to emphasize the exact names of the images so we don't burn ourselves in the future 😂 But, it's fine like this, I was also a bit confused maybe, didn't notice tabs at that moment probably